src/app/constants/time.constant.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Milliseconds in an hour
```

Comment:
I don't think the comments are needed anymore. The variable name is clear enough.